ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/deduplication/CommandDeduplicationBase.scala

```diff
@@ -330,6 +332,33 @@ private[testtool] abstract class CommandDeduplicationBase(
     }
   })
 
+  test(
+    s"${testNamingPrefix}DeduplicateParallelSubmissions",
+    "Commands submitted at the same, in parallel, should be deduplicated",
+    allocate(SingleParty),
+  )(implicit ec => { case Participants(Participant(ledger, party)) =>
+    lazy val request = ledger
+      .submitRequest(party, DummyWithAnnotation(party, "Duplicate").create.command)
+      .update(
+        _.commands.deduplicationPeriod := DeduplicationPeriod.DeduplicationTime(
+          deduplicationDuration.asProtobuf
+        )
+      )
+    val numberOfParallelRequests = 10
+    Future
+      .traverse(Seq.fill(numberOfParallelRequests)(request))(request => {
+        submitRequestAndGetStatusCode(ledger)(request, party)
+      })
+      .map(_.groupMapReduce(identity)(_ => 1)(_ + _))
+      .map(responses => {
+        val expectedDuplicateResponses = numberOfParallelRequests - 1
+        assert(
+          responses == Map(Code.ALREADY_EXISTS -> expectedDuplicateResponses, Code.OK -> 1),
```

Comment:
Please also allow `Code.ABORTED` as an error code for a duplicate command (see https://github.com/digital-asset/daml/pull/10869/files#r707999476)

Reply:
Done, we allow for ABORTED, INTERNAL, ALREADY_EXISTS